src/Imdb/Title.php

```diff
@@ -1262,13 +1262,15 @@ public function sound()
     public function mpaa()
     {
         if (empty($this->mpaas)) {
-            $this->getPage("ParentalGuide");
-            if (preg_match_all("|/search/title\?certificates=.*?>\s*(.*?):(.*?)<|", $this->page["ParentalGuide"],
-                $matches)) {
-                $cc = count($matches[0]);
-                for ($i = 0; $i < $cc; ++$i) {
-                    $this->mpaas[$matches[1][$i]] = $matches[2][$i];
-                }
+            $xpath = $this->getXpathPage("ParentalGuide");
+            if (empty($xpath)) {
+                return array();
+            } // no such page
```

Comment:
Odd place for a comment, don't think we need it.

Reply:
That is the way it was always there but sure it can be removed, the function of the if is clear.